src/sympc/protocol/default/default.py

```diff
@@ -0,0 +1,62 @@
+"""Default Protocol."""
+
+# stdlib
+from typing import Any
+from typing import Dict
+from typing import List
+
+from sympc.protocol.protocol import Protocol
+from sympc.tensor import ReplicatedSharedTensor
+from sympc.tensor.tensor import SyMPCTensor
+
+
+class DefaultProtocol(metaclass=Protocol):
+    """Default Protocol Implementation."""
+
+    """Used for Share Level static operations like distributing the shares."""
+    share_class: SyMPCTensor = ReplicatedSharedTensor
```

Comment:
Q: Could we have it `ShareTensor` (because all the operations that work are using `ShareTensor`)

Reply:
And also the `security_levels` to be only `["semi-honest"]`

Reply:
Yeah ,george  I wanted to initialize `default protocol` ,such that I could revert `RSTensor` operation when we have arbitrary number of parties,as `Falcon` is highly customized for 3PC.Shall we have it as `RSTensor` for now?.In the long run when we are implementing ops we could have both `RSTensor` and `ShareTensor`.